src/renderer/components/dock/terminal.ts

```diff
@@ -225,11 +225,6 @@ export class Terminal {
       }
     }
 
-    // Pass the event above in DOM for <Dock/> to handle common actions
-    if (!evt.defaultPrevented) {
-      this.elem.dispatchEvent(new KeyboardEvent(type, evt));
-    }
-
```

Comment:
Removing this because we moved `keydown` listener to the global scope so <Dock/> won't listen any event anymore.